web/libs/editor/tests/e2e/tests/taxonomy.test.js

```diff
@@ -34,36 +34,35 @@ Scenario('Lines overlap', async ({ I, LabelStudio, AtTaxonomy }) => {
   }
 
   I.amOnPage('/');
+  I.wait(1);
   LabelStudio.init({
     config: `
-<View>
-  <Text name="text" value="$text"/>
-  <Taxonomy name="taxonomy" toName="text">
-    <Choice value="target group">
-      <Choice value="london london london london london london london" />
-      <Choice value="long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long line"/>
-      <Choice value="not so long line"/>
-    </Choice>
-  </Taxonomy>
-</View>`,
+      <View>
+        <Text name="text" value="$text"/>
+        <Taxonomy name="taxonomy" toName="text">
+          <Choice value="target group 1">
+            <Choice value="london london london london london london london" />
+            <Choice value="long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long long line"/>
+            <Choice value="not so long line"/>
+          </Choice>
+        </Taxonomy>
+      </View>`,
     data: {
       text: 'Annotation 1',
     },
   });
-
   AtTaxonomy.clickTaxonomy();
-  AtTaxonomy.toggleGroupWithText('target group');
+  AtTaxonomy.toggleGroupWithText('target group 1');
 
   await checkOverlapAndGap('long long long', 'not so long');
 
-
   I.amOnPage('/');
   LabelStudio.init({
     config: `
 <View>
   <Text name="text" value="$text"/>
   <Taxonomy name="taxonomy" toName="text">
-    <Choice value="target group">
+    <Choice value="target group 2">
```

Comment:
The changes like that could potentially break the test logic. Target group title most probably should be the same in each config.